src/domain/accessibility/AccessbilityPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import LoadingSpinner from '../../common/components/spinner/LoadingSpinner';
+import Hero from '../../common/components/staticPageHero/StaticPageHero';
+import { useAccessibilityPagesQuery } from '../../generated/graphql';
+import useLocale from '../../hooks/useLocale';
+import sanitizeHtml from '../../util/sanitizeHtml';
+import Container from '../app/layout/Container';
+import MainContent from '../app/layout/MainContent';
+import PageWrapper from '../app/layout/PageWrapper';
+import styles from './accessibilityPage.module.scss';
+
+const AboutPage: React.FC = () => {
+  const locale = useLocale();
+  const { data, loading } = useAccessibilityPagesQuery();
+
+  const accessibilityPage = data?.accessibilityPages.data.find(
+    (page) =>
+      !page.expired &&
+      (page.contentSection?.[locale] || page.headingSection?.[locale])
+  );
+  const headingSection = accessibilityPage?.headingSection?.[locale];
+  const contentSection = accessibilityPage?.contentSection?.[locale];
+
+  return (
+    <PageWrapper
+      className={styles.accessibilityPage}
+      title="accessibility.title"
+    >
+      <MainContent offset={-70}>
+        <LoadingSpinner isLoading={loading}>
+          {headingSection && (
+            <Hero>
+              <div
+                dangerouslySetInnerHTML={{
+                  __html: sanitizeHtml(headingSection),
```

Comment:
Hmm, is this sanitization needed here? The data come from trusted server? Or are there some other security concerns?

Maybe it is better to be safe than sorry :) but the downside is performance / sanitizeHtml package size

Reply:
In this particular case the data is coming from our CMS. Same sanitize method is also used on Events page and I don't trust LinkedEvents data importers or actually source data so much. (So we need the package anyway) The data amount is not so huge so I don't see that performance is big problem here.

Reply:
Ok sounds reasonable :)